docs/guides/launch-content.md

```diff
@@ -0,0 +1,29 @@
+# Getting Started: Launching content
+
+!!! warning
+    This guide assumes that you already have obtained the content legally, RetroArch does not provide users with copyright content.
+
+You should already have both cores and content, now we just have to launch it!
+
+### Launching
+
+Go back to the Main Menu and select **Load content**. From the last guide you should have already set the default start directory, if this is the case choose **Start directory**. If you haven't set it or you are choosing other content (such as downloaded content) select the relvent option.
+
+From here it is up to you to navigate your game folder and find the game you want to try, select it once you have.
+
+!!! note
+    If the game is zipped you will be presented with two options. *Browse archive* will allow you to open it and select the contents. While *Load archive* will try and load the archive as an image. In most cases if you simply just compressed the image, either option will do.
+
+!!! note
+    RetroArch will automatically choose the appropriate core for your content based on file extension. However if two or more cores use the same file extension, it will ask you to choose the core. You should always choose the platform the game was suppose to run on.
+
+### Playing
+
+Well done, you have finally launched your first game with RetroArch!
```

Comment:
Don't need the exclamation. See Google's notes on documentation writing with exclamation points: https://developers.google.com/style/exclamation-points